Doc/library/pyexpat.rst

```diff
@@ -196,6 +196,37 @@ XMLParser Objects
    :exc:`ExpatError` to be raised with the :attr:`code` attribute set to
    ``errors.codes[errors.XML_ERROR_CANT_CHANGE_FEATURE_ONCE_PARSING]``.
 
+.. method:: xmlparser.SetReparseDeferralEnabled(enabled)
+
+.. warning::
+
+   Calling ``SetReparseDeferralEnabled(False)`` has security implications,
+   as detailed below; please make sure to understand these consequences
+   prior to using the ``SetReparseDeferralEnabled`` method.
```

Comment:
Add indentation (3 spaces) and move it to the end of SetReparseDeferralEnabled description. I am not sure that the `warning` directive is needed here, it can be just plain text, but let it be for now.

Reply:
@serhiy-storchaka moving it to the end will kill most of the effect and need adjustment of "detailed below". I can do the shift right by three space but the rest does not sound ideal really.

Reply:
@serhiy-storchaka now indented (but still on top), pushed.